libr/core/cmd_anal.c

```diff
@@ -593,6 +593,8 @@ static void core_anal_bytes(RCore *core, const ut8 *buf, int len, int nops, int
 				if (p && *p && strcmp (p, "null"))
 					r_cons_printf ("\"stack\":\"%s\",", p);
 			}
+			if (op.stackptr != 0)
```

Comment:
yep, we decided to use a different syntax and the code is being slowly updated :) thanks


> On 22 Aug 2016, at 18:04, Wladimir J. van der Laan <notifications@github.com> wrote:
> 
> In libr/core/cmd_anal.c <https://github.com/radare/radare2/pull/5582#discussion_r75706286>:
> 
> > @@ -593,6 +593,8 @@ static void core_anal_bytes(RCore *core, const ut8 *buf, int len, int nops, int
> >  				if (p && *p && strcmp (p, "null"))
> >  					r_cons_printf ("\"stack\":\"%s\",", p);
> >  			}
> > +			if (op.stackptr != 0)
> Sure - though the conditionals above here (e.g. if (hint && hint->jump != UT64_MAX)) don't have braces either, I just mimicked that.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/radare/radare2/pull/5582/files/7e4fcfc27b0fe0a9401775b31d44b2de38d8c406#r75706286>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AA3-lq2ISs8-dVvKw2BOAGtwJHTDHRwYks5qich4gaJpZM4Jp9aX>.
>